ranger/config/rc.conf

```diff
@@ -27,6 +27,10 @@
 set viewmode miller
 #set viewmode multipane
 
+#enable / disable the status or the titlebar. Allowed values are 'true' or 'false'
```

Comment:
Make this a proper sentence like the other comments. Drop the allowed values. Maybe phrase it as a question: "Show the statusbar?" Make a comment for each don't group these settings.